src/app/game-state/followers.service.ts

```diff
@@ -273,12 +277,13 @@ export class FollowersService {
       this.logService.addLogMessage("A new follower shows up, but they were a " + job + " and you don't want any more of those.","STANDARD","FOLLOWER");
       return;
     }
-
-    this.logService.addLogMessage("A new follower has come to learn at your feet.","STANDARD","FOLLOWER");
+    
+    const lifespanDivider = this.followerLifespanDoubled ? 5 : 10;
+    this.logService.addLogMessage("A new " + job + " has come to learn at your feet.","STANDARD","EVENT");
     this.followers.push({
       name: this.generateFollowerName(),
       age: 0,
-      lifespan: this.characterService.characterState.lifespan,
+      lifespan: this.characterService.characterState.lifespan / lifespanDivider,
```

Comment:
Not only reverted this overpowered change, but got an idea to create the achievement.